layers/meta-resin-beaglebone/recipes-kernel/linux/linux-beagleboard-4.4/0001-omap_hsmmc-Reduce-max_segs.patch

```diff
@@ -0,0 +1,36 @@
+From fb8ec188334f571e835115a0deb7b5be8662bb38 Mon Sep 17 00:00:00 2001
+From: Will Newton <willn@resin.io>
+Date: Fri, 16 Jun 2017 12:13:21 +0200
+Subject: [PATCH] omap_hsmmc: Reduce max_segs
+
+Reduce max_segs to a value that allows allocation of an entire
+descriptor list within a single page. This avoids doing a
+higher order GFP_ATOMIC allocation when setting up a transfer
+which can potentially fail and lead to I/O failures.
+
+Upstream-Status: Pending
```

Comment:
We also need to check and see if all the patches we added to mitigate this issue are also added to the 1.X branch and not only in 2.X